libs/execution_context/ebpf_protocol.h

```diff
@@ -35,7 +35,7 @@ typedef enum _ebpf_operation_id
     EBPF_OPERATION_GET_NEXT_MAP_ID,
     EBPF_OPERATION_GET_NEXT_PROGRAM_ID,
     EBPF_OPERATION_GET_OBJECT_INFO,
-    EBPF_OPERATION_GET_NEXT_PINNED_PROGRAM_PATH,
+    EBPF_OPERATION_GET_NEXT_PINNED_OBJECT_PATH,
```

Comment:
Please create a new message for this. The intent is for the contract between user mode and kernel mode to be backwards compatible (for servicing reasons). I would prefer an approach where we add a new message and deprecate and eventually remove the old one.

Reply:
Can do! Is this documented somewhere? I'm still a bit fuzzy what kind of guarantees will be made in terms of backwards / forwards compat etc.